src/App.tsx

```diff
@@ -1,14 +1,56 @@
-import React from 'react';
+import React, { useState, useCallback } from 'react';
+import classNames from 'classnames';
+import { debounce } from 'lodash';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState('');
+  const [applyQuery, setApplyQuery] = useState('');
+  const [selectedPerson, setSelectedPerson] = useState({
+    ...peopleFromServer[0],
+    name: 'No selected person',
+  });
+  const { name, born, died } = selectedPerson;
+  const debounceDelay = 1000;
+  const applyedQuery = useCallback(
+    debounce(setApplyQuery, debounceDelay),
+    [debounceDelay],
+  );
+
+  const visiblePeopleFn = () => {
```

Comment:
you can just wrap it with `useMemo`

Reply:
![Screenshot_1](https://github.com/mate-academy/react_autocomplete/assets/115159654/b8aad129-6dcb-4fc8-90d0-af6378502b74)
It was, but the mentor had previously written 'useMemo is redudant here'